lib/mixlib/install/backend/package_router.rb

```diff
@@ -147,16 +147,29 @@ def get(url)
           uri = URI.parse(endpoint)
           http = Net::HTTP.new(uri.host, uri.port)
           http.use_ssl = (uri.scheme == "https")
-
           full_path = File.join(uri.path, url)
-          request = Net::HTTP::Get.new(full_path)
 
-          res = http.request(request)
+          res = http.request(create_http_request(full_path))
 
           res.value
           JSON.parse(res.body)
         end
 
+        def create_http_request(full_path)
+          require "mixlib/install/version"
+
+          request = Net::HTTP::Get.new(full_path)
+          request.add_field("User-Agent", "mixlib-install/#{Mixlib::Install::VERSION}")
+
+          if options.user_agent_headers
+            options.user_agent_headers.each do |agent|
+              request.add_field("User-Agent", agent)
```

Comment:
So [per the RFC](https://tools.ietf.org/html/rfc1945#page-46) we don't want 1..n `User-Agent` request headers but instead want a single `User-Agent` with multiple values delimited with a space.